utils/misc.mli

```diff
@@ -118,8 +139,9 @@ val replace_substring: before:string -> after:string -> string -> string
            occurences of [before] with [after] in [str] and returns
            the resulting string. *)
 
-val rev_split_words: string -> string list
-        (* [rev_split_words s] splits [s] in blank-separated words, and return
+val rev_split_words: ?separator:char -> string -> string list
```

Comment:
I can change it to a list if you prefer, although I would still argue it basically doesn't matter in this case.